build.gradle

```diff
@@ -32,11 +33,10 @@ repositories {
 }
 
 dependencies {
-    compile 'commons-codec:commons-codec:1.5'
-    compile 'org.apache.httpcomponents:httpclient:4.5.3'
-    compile 'org.apache.httpcomponents:fluent-hc:4.5.3'
+    compile 'com.squareup.okhttp3:okhttp:3.9.1'
+    compile 'com.squareup.okhttp3:mockwebserver:3.9.1'
```

Comment:
testCompile. Leaving it as compile makes proguard scream and my build fails

Reply:
Fixed, thanks!